jsonschema2pojo-integration-tests/src/test/java/org/jsonschema2pojo/integration/DynamicPropertiesIT.java

```diff
@@ -45,6 +45,8 @@ public void shouldSetStringField() throws Throwable {
 
     @Test
     public void shouldSetStringFieldJava7() throws Throwable {
+        // jdk21 cannot build 1.7 target
```

Comment:
It seems like we should use the DisabledForJreRange JUnit annotation to turn tests like this off when needed.

Reply:
This assumeTrue is probably fine.  I am having trouble figuring out where `JAVA_EARLIER_21` is coming from.

Reply:
DisabledForJreRange is a better way to do this since jupiter. That's for the highlight.